app/Http/Controllers/StatisticsController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Support\Facades\DB;
+
+class StatisticsController extends Controller
+{
+    public function index()
+    {
+        $elephpants = DB::table('elephpants')
+            ->select(DB::raw('COUNT(elephpant_user.elephpant_id) as nbElephpant, name, description, SUM(elephpant_user.quantity) as totalElephpant'))
+            ->leftJoin('elephpant_user', 'elephpants.id', '=', 'elephpant_user.elephpant_id')
+            ->orderBy('nbElephpant', 'desc')
+            ->orderBy('elephpants.id', 'desc')
+            ->orderBy('totalElephpant', 'desc')
+            ->groupBy('elephpants.id')
+            ->get();
+
+        $nbUsers = User::query()->count();
```

Comment:
just one question about this. here we are considering users that has no elephpants too. shouldn't we consider users with at least one elephpant? using `elephpant_user.user_id`?